### ISSUE TITLE
Misalignment of Inbox when reading pane turned on with no split

### ISSUE THREAD

**Issue by aaronblair** [leggett/simplify#519]
See below image. Fixes when reading pane turned off in settings. 

<img width="989" alt="image" src="https://user-images.githubusercontent.com/1298050/107862418-fb037180-6e09-11eb-942b-3badee0adf53.png">

**Comment by leggett** [comment]
Hi Aaron -- Thanks for the report. I'm not able to reproduce this issue though. :( 

Can you send a system report (via email is fine)? If you go to Gmail and hit Alt+I (⌥I on Mac) it will include a system config will tell me more about your setup.

**Comment by aaronblair** [comment]
I just emailed it to you. I also noticed that it only happens in Firefox (not Chrome).

**Comment by leggett** [comment]
Here is the config:

Simplify v.2.2.2 - Configuration: simplify highDensity dateGroup izBgDefault minimizeChat invertAddons hideTabIcons composeActions ca3 caI caL cfC cfO cfU cfQ cfR aAX navOpen newUI mediumTheme inInbox inList nPane - Browser: 5.0 (Macintosh) - Window: 1440x875 (100pct zoom) - Language: en

**Comment by leggett** [comment]
Looking at this on Firefox and I can't seem to reproduce the issue. What version of Firefox are you running? 

Here is what I'm getting with the same setup:
<img width="1440" alt="Screen Shot 2021-02-14 at 12 45 33 PM" src="https://user-images.githubusercontent.com/8763662/107888665-a4a63980-6ec2-11eb-9008-32500d9a6365.png">

**Comment by leggett** [comment]
It is 1px off. And I see more oddities when I open quick settings or an add-on. I will fix those things.

**Comment by aaronblair** [comment]
I tried removing all my extensions and nuking my browser cache/settings, doesn't seem to have made any difference. It must be a combination of other settings in Gmail that is causing it.

<img width="642" alt="image" src="https://user-images.githubusercontent.com/1298050/107889199-26e82b00-6ece-11eb-9422-0f987611fa71.png">

**Comment by aaronblair** [comment]
I'm not sure if it's at all related, but when going from the inbox to opening an email (or vice-versa), there is a quick jostle as it renders. I assume this is a limitation of being a browser extension. It not a big deal, but it's noticeable. 

PS. The v2 implementation is incredible. I'm shocked that Gmail itself doesn't commit to a simpler interface. It's shocking how much of a difference it makes in usability.

**Comment by leggett** [comment]
Yeah, I've tried to get rid of as much of the jostle as possible. I've even recorded the transition with the inspector open and looked at how things unfold within the <1 second transition. 

It is mostly caused by (1) elements and classnames change before or after the url is updated which is what triggers the transition and (2) the list and message view being different widths and sharing the action bar. I've tried triggering off of other changes but it is a lot more brittle and didn't get rid of the jostle.
